app/workers/rapid_ready_for_decision/disability_compensation_job.rb

```diff
@@ -46,6 +46,13 @@ class AccountNotFoundError < StandardError; end
 
     private
 
+    def merge_patient_info(form526_submission)
+      {
+        **form526_submission.full_name,
+        va_eauth_birthdate: form526_submission.auth_headers['va_eauth_birthdate']
+      }
```

Comment:
This is more clear to me:
```suggestion
      form526_submission.full_name.merge(
        va_eauth_birthdate: form526_submission.auth_headers['va_eauth_birthdate']
      )
```
or maybe
```suggestion
      form526_submission.full_name.merge(form526_submission.auth_headers.slice('va_eauth_birthdate'))
```

Reply:
I see. `#merge` returns a new hash. Thanks!